src/components/action-list/ActionList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ActionListHole from './ActionListHole';
+
+const DIRECTION = {
+  TO_RIGHT: 'to-right',
+  TO_TOP: 'to-top',
+  CENTERED: 'centered',
+  SPACE_BETWEEN: 'space-between'
+};
+
+const ActionList = ({holes = [], direction, noWrap}) => {
+  const actionListClass = classNames('sg-actions-list', {
+    [`sg-actions-list--${direction}`]: direction,
+    'sg-actions-list--no-wrap': noWrap
+  });
+
+  return (
+    <div className={actionListClass}>
+      {holes.map((hole, i) => <ActionListHole key={i}>{hole}</ActionListHole>)}
```

Comment:
Maaati this does not change a thing
just put children here and push this list from outside
Or check if childrea are type of <ActionListHole> if not the map like above